### PR TITLE
Fixes #10443

### LINKED ISSUE #10443: Error in 16 beta: undefined is not a function (evaluating 'owner.getName()')
This line is wrong in Fiber and crashes:

https://github.com/facebook/react/blob/58e6edeef61f5ab5b81530820571213161940d6a/src/renderers/native/findNodeHandle.js#L79

It should be changed to use getComponentName(owner).

### PR DESCRIPTION
**Pull request by fxfactorial**

### CODE REVIEW

File: src/renderers/native/findNodeHandle.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ var ReactInstanceMap = require('ReactInstanceMap');
 var ReactNativeFeatureFlags = require('ReactNativeFeatureFlags');
 var ReactNativeFiberRenderer = require('ReactNativeFiberRenderer');
 var {ReactCurrentOwner} = require('ReactGlobalSharedState');
-
+var getComponentName = require('getComponentName.js');
```
Comment (acdlite) [inline src/renderers/native/findNodeHandle.js]: Remove the `.js` extension

File: src/renderers/native/findNodeHandle.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ function findNodeHandle(componentOrHandle: any): any {
           'never access something that requires stale data from the previous ' +
           'render, such as refs. Move this logic to componentDidMount and ' +
           'componentDidUpdate instead.',
-        owner.getName() || 'A component',
+        getComponentName(owner)
```
Comment (acdlite) [inline src/renderers/native/findNodeHandle.js]: This should be

```js
getComponentName(owner) || 'A component',
```

**Comment by fxfactorial** [comment]
Done, I forgot that FB prefers trailing `,`, force pushed

**Comment by fxfactorial** [comment]
@acdlite I think that should cover it.

**Comment by acdlite** [comment]
Thanks!

**Comment by gaearon** [comment]
This is not enough to fix the issue. Please see https://github.com/facebook/react/issues/10518.

**Pull request merged**